wgsl/index.bs

```diff
@@ -3940,6 +3940,13 @@ The result type depends on the number of letters provided. Assuming a `vec4<f32>
 
 #### Component reference from vector reference #### {#component-reference-from-vector-reference}
 
+A [=write access=] to component of a vector **may** access all of the [=memory
```

Comment:
I'm not sure this is the ideal spec section to specify this, but it felt better than specifying it in the memory access mode, memory locations or memory layout sections.

Reply:
This is as good a place as any, until https://github.com/gpuweb/gpuweb/pull/2055/ lands.  
When that lands, its normative spot would be around the statement:

> A memory operation accesses exactly the set of [=memory location|locations=]
associated with the particular [=memory view=] used in the operation. 

Because the two statements are in conflict, and the exception overrules the general case.